pkg/genericapiserver/endpoints/handlers/rest.go

```diff
@@ -140,7 +140,7 @@ func GetResource(r rest.Getter, e rest.Exporter, scope RequestScope) restful.Rou
 			options := metav1.GetOptions{}
 			if values := req.Request.URL.Query(); len(values) > 0 {
 				exports := metav1.ExportOptions{}
-				if err := scope.ParameterCodec.DecodeParameters(values, schema.GroupVersion{Version: "v1"}, &exports); err != nil {
+				if err := metainternalversion.ParameterCodec.DecodeParameters(values, scope.MetaGroupVersion, &exports); err != nil {
```

Comment:
genericapiserver can handle any meta group defined.  We only have one today, but if we added a different group, then a server that supports meta/v2 would need the decoding.